README.md

```diff
@@ -132,7 +130,7 @@ Thank you for downloading and using the PrestaShop Open Source e-commerce soluti
 [available-features]: https://www.prestashop.com/en/online-store-builder
 [download]: https://www.prestashop.com/en/download
 [forums]: https://www.prestashop.com/forums/
-[chat]: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md
+[chat]: https://www.prestashop-project.org/slack/
 [user-doc]: https://doc.prestashop.com
 [contributing-md]: CONTRIBUTING.md
 [contributing-tutorial]: https://devdocs.prestashop.com/1.7/contribute/
```

Comment:
Could you update links withn`8.0` version ?

Reply:
This raises a question which is: what should the README page of GitHub display to visitors ?

Current latest stable release of PS is 1.7.7.8 . Next RC is 1.7.8.0 . 8.0.0 is far away from being complete.

So just like on https://devdocs.prestashop.com/ we have "1.7" as default and "8" as dev, I think we should present in the README the information about the latest stable version (so basically what is running in production) to visitors, not the one that will come many months later.

Reply:
We are on the develop branch, so it should target the develop version. Otherwise, if we continue targeting the 1.7, the code display is no longer relevant with links.
I agree with @Progi1984

Reply:
> We are on the develop branch, so it should target the develop version. Otherwise, if we continue targeting the 1.7, the code display is no longer relevant with links.

Unfortunately, when you visit https://github.com/prestashop/prestashop GitHub prompts the `develop` README. So GitHub prompts the README of the being-developed version rather than the current one.

I wish we could prompt the README of `1.7.8.x` branch 🤷  to visitors that would solve the issue.

Reply:
The link should be `https://devdocs.prestashop.com/latest/contribute/` otherwise

Reply:
Not willing to fight you so I did the changes you requested.
https://github.com/PrestaShop/PrestaShop/pull/25923/commits/1b3996da41944fdea85455329f19a9310b37ed26